master/internal/config/ui_config.go

```diff
@@ -0,0 +1,101 @@
+package config
+
+import (
+	"os"
+
+	"github.com/pkg/errors"
+
+	"github.com/determined-ai/determined/master/internal/license"
+)
+
+// MediaAssetVariations allow variations of a media asset to be defined.
+type MediaAssetVariations struct {
+	DarkHorizontal  string `json:"dark_horizontal"`
+	DarkVeritical   string `json:"dark_vertical"`
+	LightHorizontal string `json:"light_horizontal"`
+	LightVeritical  string `json:"light_vertical"`
+}
```

Comment:
I'd probably take out the vertical and horizontal distinctions. looking at our usage of those for the Logo I don't think we're using the two variations and where we do I think it's confusing.